### DISCUSSION TITLE
Example : fetch data and auto populate in forms

### DISCUSSION THREAD

**Discussion by chann44** [pmndrs/jotai#1575] [General]
can we add a example in Examples where fetching data from a source and auto population of multi step form

**Comment by dai-shi** [comment]
Please someone try it!

**Reply by chann44** [reply]
i am building a similar app so i thought i would use this to build a example out of it

**Reply by chann44** [reply]
so i might make a pr in examples was just asking if it's wort to add in examples or not

**Reply by dai-shi** [reply]
Oh, I misunderstood the description.
I'm not sure if it's suitable in examples (if it's too complicated). But, we are interested in adding best practices in docs.

**Reply by chann44** [reply]
let's try it and then see if it's short and can be used i'll make  a pr for examples
